tiledb/bioimg/converters/base.py

```diff
@@ -246,7 +250,12 @@ def tile_to_tiledb(level_tile: Tuple[slice, ...]) -> None:
 
             # Write group metadata
             with tiledb.Group(output_path, "w") as group:
-                group.meta.update(reader.group_metadata, axes=input_axes.dims)
+                group.meta.update(
+                    reader.group_metadata,
+                    axes=input_axes.dims,
+                    pckg_version=version,
```

Comment:
Is this deliberately spelled `pckg` or a typo for `pkg`?

Reply:
It was deliberately but let's use a 3-letter abbreviation.